lib/senders/email.js

```diff
@@ -337,10 +337,11 @@ module.exports = function (log, config) {
           mailer.sendMail(emailConfig, (err, status) => {
             if (err) {
               log.error({
-                op: 'mailer.send.1',
-                err: err && err.message,
-                status: status && status.message,
-                id: status && status.messageId,
+                op: 'mailer.send.error',
```

Comment:
I changed the `op` here to differentiate it from the success log message, which had the same `op`.